Installation-SUSE-CAP-1.0.adoc

```diff
@@ -0,0 +1,116 @@
+    CLUSTER_ADMIN_PASSWORD: changeme
+
+    # Domain for SCF. DNS for *.DOMAIN must point to a kube node's (not master)
+    # external ip address.
+    DOMAIN: cf-dev.io
+
+    # Password for SCF to authenticate with UAA
+    UAA_ADMIN_CLIENT_SECRET: uaa-admin-client-secret
+
+    # UAA host/port that SCF will talk to. If you have a custom UAA
+    # provide its host and port here. If you are using the UAA that comes
+    # with the SCF distribution, simply use the two values below and
+    # substitute the cf-dev.io for your DOMAIN used above.
+    UAA_HOST: uaa.cf-dev.io
+    UAA_PORT: 2793
+kube:
+    # The IP address assigned to the kube node pointed to by the domain.
+    external_ip: 192.168.77.77
+    storage_class:
+        # Make sure to change the value in here to whatever storage class you use
+        persistent: "persistent"
+        shared: "shared"
+    # The registry the images will be fetched from. The values below should work for
+    # a default installation from the suse registry.
+    registry:
+       hostname: "registry.suse.com"
+       username: ""
+       password: ""
+    organization: "cap"
+
+    # The next line is needed for CaaS Platform 2, but should _not_ be there for CaaS Platform 1
+    auth: rbac
+----
+
+If you are deploying to a CaaS Platform with SES you need to prepare the environment using these commands:
+
+[source,bash]
+----
+$ kubectl create namespace uaa
+$ kubectl get secret ceph-secret-admin -o json --namespace default | sed 's/"namespace": "default"/"namespace": "uaa"/' | kubectl create -f -
+$ kubectl create namespace scf
+$ kubectl get secret ceph-secret-admin -o json --namespace default | sed 's/"namespace": "default"/"namespace": "scf"/' | kubectl create -f -
+$ kubectl create namespace stratos
+$ kubectl get secret ceph-secret-admin -o json --namespace default | sed 's/"namespace": "default"/"namespace": "stratos"/' | kubectl create -f -
+----
+
+=== Deploy using Helm
+
+*If you haven't installed the helm client follow the instructions at https://docs.helm.sh/using_helm/#quickstart*
+
+==== Add the CAP Helm Chart repository:
+
+[source,bash]
+----
+$ helm repo add suse https://kubernetes-charts.suse.com/
+----
+
+==== Deploy Cloud Foundry
+
+. Deploy UAA
+[source,bash]
+$ helm install suse/uaa \
+       --namespace uaa \
+       --values scf-config-values.yaml
+
+. Wait until the `uaa:secret-generator` pod was run and disappeared:
+[source,bash]
+$ watch -c 'kubectl get pods --all-namespaces'
+# Then copy the UAA CA Cert to later give to SCF
+$ CA_CERT="$(kubectl get secret secret --namespace uaa -o jsonpath="{.data['internal-ca-cert']}" | base64 --decode -)"
+
+. With UAA deployed, use Helm to deploy SCF.
+[source,bash]
+$ helm install suse/cf \
+       --namespace scf \
+       --values scf-config-values.yaml \
+       --set "env.UAA_CA_CERT=${CA_CERT}"
+
+. Wait for everything to be ready:
+[source,bash]
+$ watch -c 'kubectl get pods --all-namespaces'
+
```

Comment:
Here it's also possible to keep the indent just with a "+":
```
[source,bash]
$ watch -c 'kubectl get pods --all-namespaces'
+
Stop watching when all pods show state `Running` and Ready is `n/n` (instead of `k/n`, `k < n`).
```